src/docs/implementations/instana.adoc

```diff
@@ -0,0 +1,24 @@
+= Micrometer Instana
+Fabian Lange <fabian.lange@instana.com>
+:toc:
+:sectnums:
+:system: instana
+
+Instana is an automatic application performance management and infrastructure monitoring system.
+
+== Installation and Configuration
+
+Instana will automatically detect and report all metrics without the need of any additional dependency or configuration!
```

Comment:
Does Instana support collecting metrics from any `MeterRegistry` implementation? It may help users to understand if it is mentioned explicitly. I was left wondering what implementation I should use since there is no `InstanaMeterRegistry` that I'm aware of. At a minimum, if a user wants to publish metrics only to Instana, they need to use a `SimpleMeterRegistry`, I suppose? Does Instana handle if more than one `MeterRegistry` is configured (including perhaps a `CompositeMeterRegistry`)?

Reply:
Good question @shakuzen - I will make this more explicit.

Yes we automatically find any subclass instance of `MeterRegistry` and collect all the metrics that are available on them. Instana will use all available instances.
Actually I would need to check what it will exactly do with a `CompositeMeterRegistry`, but I think it does work as expected, as the `MeterRegistry` is just the means to access the individual `Meter`s